blueprints/component-test/index.js

```diff
@@ -86,21 +86,21 @@ module.exports = useTestFrameworkDetector({
     };
   },
 
-  _useSeparateInlinePrecompileAddon() {
+  _useNamedHbsImport() {
     let htmlbarsAddon = this.project.addons.find(a => a.name === 'ember-cli-htmlbars');
 
-    if (htmlbarsAddon === undefined) {
+    if (htmlbarsAddon && semver.gte(htmlbarsAddon.pkg.version, '4.0.0-alpha.1')) {
       return true;
-    } else if (semver.gte(htmlbarsAddon.pkg.version, '4.0.0-alpha.1')) {
```

Comment:
This line implies that, if the version is `ember-cli-htmlbars` is 3.x or less, we will still use the named import (because we are returning `undefined`). Could you verify if this was intended or a bug?

Reply:
3.x should use `import hbs from 'htmlbars-inline-precompile'`, which AFAICT is still what is done after this change.